oss_fuzz_integration/jvm.patch

```diff
@@ -0,0 +1,47 @@
+# Packing for fuzz-introspector
```

Comment:
I'd prefer to put this logic into the OSS-Fuzz environment rather, i.e. having it as a script we run in the base-builder